controllers/articleController.js

```diff
@@ -7,4 +7,14 @@ exports.getArticleByID = (req, res, next) => {
     res.status(200).send({ article });
   })
   .catch(next);
+};
+
+exports.patchArticleByID = (req, res, next) => {
+  const body = req.body;
```

Comment:
(subjective point)
Try to be consistent in the way you are using destructuring 

You are also passing a number and the whole `body` object down to your model
It would make more sense to only pass the `inc_votes` down